### PR TITLE
Added "EXPLODE" operator

### PR DESCRIPTION
**Pull request by llxxll12345**
Finished writing "EXPLODE" operator. With Unnest planner and executor added. Tests yet to be added.

### CODE REVIEW

**Comment by xzdandy** [comment]
Do not add eva_dataset into the repo.

**Review by gaurav274 — CHANGES_REQUESTED** [review]
1. Remove the binary files.
2. Replace explode with unnest.

I'll add more comments for other edits.

File: src/parser/evaql/evaql_lexer.g4 [hunk 1]
```diff
@@ -36,6 +36,9 @@ DROP:                                'DROP';
 EXIT:                                'EXIT';
 EXISTS:                              'EXISTS';
 EXPLAIN:                             'EXPLAIN';
+
+EXPLODE:                             'EXPLODE';
```
[the remark this answers is not in the document]
Comment (llxxll12345) [reply src/parser/evaql/evaql_lexer.g4:40]: choose to stick with explode now.

**Comment by jiashenC** [comment]
I think even though the file is deleted, it will still be stored in the git history tree [some references](https://www.deployhq.com/git/faqs/removing-large-files-from-git-history), which slows down other commits. Probably we need to filter out those histories.

**Comment by xzdandy** [comment]
> I think even though the file is deleted, it will still be stored in the git history tree [some references](https://www.deployhq.com/git/faqs/removing-large-files-from-git-history), which slows down other commits. Probably we need to filter out those histories.

That's a good point. @llxxll12345 Check if you are able to filter out those histories.

**Pull request closed without merging**